lib/wallet/http.js

```diff
@@ -1084,6 +1084,45 @@ class HTTP extends Server {
       return res.json(200, mtx.getJSON(this.network));
     });
 
+    // Create auction-related transactions in advance (bid and reveal for now)
+    this.post('/wallet/:id/auction', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      const name = valid.str('name');
+      const bid = valid.u64('bid');
+      const lockup = valid.u64('lockup');
+      const passphrase = valid.str('passphrase');
+      const sign = valid.bool('sign', true);
+      const broadcastBid = valid.bool('broadcastBid');
+
+      assert(name, 'Name is required.');
+      assert(bid != null, 'Bid is required.');
+      assert(lockup != null, 'Lockup is required.');
+      assert(broadcastBid != null, 'broadcastBid is required');
```

Comment:
micro-nit: missing period at the end (all the other error statements have it)

Reply:
Had to re-read it two times to spot it --"